GitUI/CommandsDialogs/RevisionFileTreeControl.Designer.cs

```diff
@@ -339,6 +341,15 @@ private void InitializeComponent()
             this.FileText.Size = new System.Drawing.Size(487, 303);
             this.FileText.TabIndex = 0;
             // 
+            // BlameControl
+            // 
+            this.BlameControl.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.BlameControl.Location = new System.Drawing.Point(0, 0);
+            this.BlameControl.Margin = new System.Windows.Forms.Padding(0);
+            this.BlameControl.Name = "blameControl";
+            this.BlameControl.Size = new System.Drawing.Size(487, 303);
+            this.BlameControl.TabIndex = 0;
```

Comment:
Conflict with `FileText.TabIndex` (although they are not displayed at the same time).

Reply:
That was intentional, really a need to separate them?

Reply:
As a general rule: no item should have identical indices. Though this is the keyboard tab order, i.e. used when tab between controls. Accessibility tools don't use it (AFAIK), and use the order in which controls are added into respective parents instead.

Reply:
Do you request to change this?